bundle/jsky.app.ot/src/main/scala/edu/gemini/catalog/ui/image/BackgroundImageLoader.scala

```diff
@@ -76,12 +77,22 @@ object BackgroundImageLoader {
 
   /** Called when a program is created to download its images */
   def watch(prog: ISPProgram): Unit = {
+    // At startup only load images for active programs
+    def needsImage(ctx: TpeContext): Boolean = {
+      ctx.obsShell.exists(ObservationStatus.computeFor(_) match {
+        case ObservationStatus.INACTIVE | ObservationStatus.OBSERVED => false
+        case _                                                       => true
```

Comment:
Here, it seems like it might be more logical to use the `ObservationStatus.isActive` return value instead of matching? Seems to correspond to exactly what you need, and I suspect that that is the sensible value you'd want. So:
`ctx.obsShell.exists(ObservationStatus.computeFor(_).isActive)`?